common/test/acceptance/tests/lms/test_lms_instructor_dashboard.py

```diff
@@ -60,8 +60,8 @@ def setUp(self):
     def test_instructor_dashboard_a11y(self):
         self.instructor_dashboard_page.a11y_audit.config.set_rules({
             "ignore": [
+                'checkboxgroup',  # TODO: AC-552
```

Comment:
It was triggering a failure, but isn't part of this scope of work. I've got another PR that will address it.